recipe/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "tensorflow-metadata" %}
+{% set version = "0.24.0" %}
+# This is the recipe for tensorflow-metadata
+package:
+   name: {{ name }}
+   version: {{ version }}
+
+source:
+  git_url: https://github.com/tensorflow/metadata.git
+  git_rev: v{{ version }}
+  
+build:
+  number: 1
+  noarch: python
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - bazel {{ bazel }}
+    - python
+  run:
+    - python 
+#    - absl-py>=0.9,<0.11
```

Comment:
Tensorflow-metadata needs absl_py >=0.9 but we are at 0.8.1 level. absl_py is widely used by many packages so I need to ensure if all the packages can work with it.